src/fromager/packagesettings.py

```diff
@@ -638,6 +638,10 @@ def build_tag(self, version: Version) -> BuildTag:
         suffix = ""
         return release, suffix
 
+    def get_changelog(self, version: Version) -> list[str]:
+        pv = typing.cast(PackageVersion, version)
+        return self._ps.changelog.get(pv, [])
```

Comment:
Sorry, I just noticed this. If you look at lines 633 and 634, you'll see that the build_tag is computed by combining the variant changelog and the package changelog. If we're returning a list here so we can log it, we should combine the same values.

Then in `build_tag`, we could use this method to get the changelog and just take the length of that result.

Reply:
Ah, I see okay! I will push the updated code

Reply:
I have pushed latest code